src/BlazorWebView/src/Wpf/BlazorWebView.cs

```diff
@@ -57,6 +57,23 @@ public class BlazorWebView : Control, IAsyncDisposable
 			name: nameof(UrlLoading),
 			propertyType: typeof(EventHandler<UrlLoadingEventArgs>),
 			ownerType: typeof(BlazorWebView));
+
+		/// <summary>
+		/// The backing store for the <see cref="BlazorWebViewInitializing"/> property.
+		/// </summary>
+		public static readonly DependencyProperty BlazorWebViewInitializingProperty = DependencyProperty.Register(
+			name: nameof(BlazorWebViewInitializing),
+			propertyType: typeof(EventHandler<BlazorWebViewInitializingEventArgs>),
+			ownerType: typeof(BlazorWebView));
+
+		/// <summary>
+		/// The backing store for the <see cref="BlazorWebViewInitialized"/> property.
```

Comment:
```suggestion
		/// The backing store for the <see cref="BlazorWebViewInitialized"/> event.
```

And since you're here, can you fix the doc comment for the UrlLoading backing event too?